Pages: 123 Next Last   
* The best metric field is obtained from the min/max of primary metric achieved by a run

Waiting for metrics...
No child metric is found
Click here to see the run in Azure Machine Learning studio
RunId: HD_5fa334d0-7ec2-4ac3-a262-1e187594feb0
Web View: https://ml.azure.com/experiments/HyperDrive-Experiment/runs/HD_5fa334d0-7ec2-4ac3-a262-1e187594feb0?wsid=/subscriptions/610d6e37-4747-4a20-80eb-3aad70a55f43/resourcegroups/aml-quickstarts-136159/workspaces/quick-starts-ws-136159

Streaming azureml-logs/hyperdrive.txt
=====================================

"<START>[2021-01-26T20:08:41.577936][API][INFO]Experiment created<END>\n"<START>[2021-01-26T20:08:42.9400735Z][SCHEDULER][INFO]The execution environment is being prepared. Please be patient as it can take a few minutes.<END>"<START>[2021-01-26T20:08:45.611444][GENERATOR][INFO]Trying to sample '15' jobs from the hyperparameter space<END>\n""<START>[2021-01-26T20:08:46.017534][GENERATOR][INFO]Successfully sampled '15' jobs, they will soon be submitted to the execution target.<END>\n"

Execution Summary
=================
RunId: HD_5fa334d0-7ec2-4ac3-a262-1e187594feb0
Web View: https://ml.azure.com/experiments/HyperDrive-Experiment/runs/HD_5fa334d0-7ec2-4ac3-a262-1e187594feb0?wsid=/subscriptions/610d6e37-4747-4a20-80eb-3aad70a55f43/resourcegroups/aml-quickstarts-136159/workspaces/quick-starts-ws-136159

Warnings:
{
  "error": {
    "code": "UserError",
    "severity": null,
    "message": "User errors were found in at least one of the child runs.",
    "messageFormat": null,
    "messageParameters": {},
    "referenceCode": null,
    "detailsUri": null,
    "target": null,
    "details": [],
    "innerError": null,
    "debugInfo": null
  },
  "correlation": null,
  "environment": null,
  "location": null,
  "time": "0001-01-01T00:00:00+00:00",
  "componentName": null
}


---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
<ipython-input-18-fc2667463e89> in <module>
      2 RunDetails(hyperdrive_run).show()
      3 hyperdrive_run.wait_for_completion(show_output = True)
----> 4 assert(hyperdrive_run.get_status() == "Completed")

AssertionError: 